anaUltraScurve.py

```diff
@@ -249,9 +321,8 @@ def overlay_fit(VFAT, CHAN):
             noise[0] = param1
             fitENC.append(vToQm*param1*options.ztrim)
             pedestal[0] = param2
-            if noise[0] > 20.0 or ped_eff[0] > 50.0: mask[0] = True
```

Comment:
Since @cbravo135 is now on holiday for ~3 weeks I would suggest to reimplement this line and open a separate issue/feature request where we address the concerns from both @cbravo135 and @lmoureaux.